app/containers/LanguageProvider/index.js

```diff
@@ -17,7 +17,7 @@ import { makeSelectLocale } from './selectors';
 export class LanguageProvider extends React.PureComponent { // eslint-disable-line react/prefer-stateless-function
   render() {
     return (
-      <IntlProvider locale={this.props.locale} key={this.props.locale} messages={this.props.messages[this.props.locale]}>
+      <IntlProvider locale={this.props.locale} key={this.props.locale} messages={this.props.messages[this.props.locale]} textComponent={React.Fragment}>
```

Comment:
cool trick, thanks for taking the time to contribute and share :)

please remember to update the corresponding template file...

https://github.com/react-boilerplate/react-boilerplate/blob/master/internals/templates/containers/LanguageProvider/index.js

Reply:
I've updated the internals LanguageProvider template file with the change and force pushed the rebased / amended commit to my repo (hoping no one is working off of this branch as of yet!).

While looking through the test however,  I felt like changing the <ConnectedLanguageProvider /> test to look for a string, is probably better than changing the <LanguageProvider /> test. Reason being <LanguageProvider /> is normally used by wrapping it around the entire App, so I would think that it should always expect ReactElements and not a string as it children. That said, when I went to go update the <ConnectedLanguageProvider> test, I went down a bunny hole where I think I found that the `store` variable isn't really being set by `configureStore`, and `browserHistory` was something that was used in React Router v3 and not recommended in v4? So, because its a little over my head for now (kind of still starting out with the React ecosystem, but learning fast because of wonderful folks like you!), I left the tests untouched. =(

Not sure if you have any recommendations / suggestions on how to go forward with this one. 

In any case, here's the trick I found to remove span tags. Thanks for looking into it!!

**A bit of warning however, I think React Fragments were only introduced / "stable" as of React v16.2.0.**

Reply:
Also, should `defineMessages()` be used to set `messages` in the tests? I thought that `defineMessages()` was mainly used to let `babel-plugin-react-intl` know one of the places scrape throughout the App for Intl Messages. Is `babel-plugin-react-intl` being invoked anywhere in the larger test process being ran by `npm test`?